README.md

```diff
@@ -19,65 +15,65 @@ cookbooks maintained by [Medidata Solutions][]:
 
 
 Requirements
-============
-
-Chef 11.12.2+
-
-This cookbook assumes you will be running a ZooKeeper ensemble for production 
-use of Mesos.
+------------
 
-The following cookbooks are dependencies:
-
-* apt
-* yum
-* java
-* docker
-
-The following cookbooks are suggested:
-
-* zookeeper (used for discovering ZooKeeper ensembles via [Netflix Exhibitor][])
-
-## Platform:
+- Chef 11
```

Comment:
No but test kitchen is using Chef 12.0.3, so maybe Chef >= 11 or something like that?